src/main/controllers/CreateThirdPartyUserSuccessController.ts

```diff
@@ -0,0 +1,22 @@
+import { PipRequest } from '../models/request/PipRequest';
+import { Response } from 'express';
+import { cloneDeep } from 'lodash';
+
+export default class CreateThirdPartyUserSuccessController {
+    public get(req: PipRequest, res: Response): void {
+        const formData = req.cookies?.formCookie ? JSON.parse(req.cookies['formCookie']) : {};
+        res.render('create-third-party-user-success', {
+            ...cloneDeep(req.i18n.getDataByLanguage(req.lng)['create-third-party-user-success']),
+            formData,
+        });
+
+        CreateThirdPartyUserSuccessController.removeThirdPartyUserFromFormCookie(res, formData);
+    }
+
+    private static removeThirdPartyUserFromFormCookie(res, formData) {
```

Comment:
Rather than specifically removing the values, could we clear the cookie instead?

Reply:
The formCookie is used for other things too in the application. I was just being a bit cautious ant not wanting to delete other information stored there for other pages that might still be required when we revisit those pages. Can clear the cookie here if u think that's ok

Reply:
I have now cleared the cookie after third party user successfully created